products/issue_tracker/backend/models.py

```diff
@@ -0,0 +1,184 @@
+from django.db import models
+from django.utils import timezone
+import uuid
+
+
+class Issue(models.Model):
+    class Status(models.TextChoices):
+        BACKLOG = "backlog", "Backlog"
+        TODO = "todo", "To Do"
+        IN_PROGRESS = "in_progress", "In Progress"
+        TESTING = "testing", "Testing"
+        DONE = "done", "Done"
+
+    class OriginProduct(models.TextChoices):
+        ERROR_TRACKING = "error_tracking", "Error Tracking"
+        EVAL_CLUSTERS = "eval_clusters", "Eval Clusters"
+        USER_CREATED = "user_created", "User Created"
+        SUPPORT_QUEUE = "support_queue", "Support Queue"
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    team = models.ForeignKey("posthog.Team", on_delete=models.CASCADE)
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    status = models.CharField(max_length=20, choices=Status.choices, default=Status.BACKLOG)
+    origin_product = models.CharField(max_length=20, choices=OriginProduct.choices)
+    position = models.IntegerField(default=0)
+
+    # Repository configuration
+    github_integration = models.ForeignKey(
+        "posthog.Integration",
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        limit_choices_to={'kind': 'github'},
+        help_text="GitHub integration for this issue"
+    )
+    
+    repository_config = models.JSONField(
+        default=dict,
+        help_text="Repository configuration with organization and repository fields"
+    )
+    
+    # GitHub integration fields (issue-specific) - kept for backward compatibility
```

Comment:
Backwards compat with what? Given this is unreleased I'd vote for simplifying things first and nuking any existing data that is invalid